src/views/incident/IncidentDetailView.tsx

```diff
@@ -29,9 +29,7 @@ export const IncidentDetailsView: React.FC<IncidentDetailsViewPropsType> = () =>
         displayAlert(`Failed to get incident with pk=${pk}`, "error");
         setIncident(null);
       });
-    // FIXME:
-    // eslint-disable-next-line
-  }, [pk]);
+  }, [pk, displayAlert]);
```

Comment:
Where does this `displayAlert` come from? does not seem to be relevant to any comments?

Reply:
That's what `FIXME` refers to. Without _displayAlert_ in dependencies, the program will not compile when `// eslint-disable-next-line` is removed

Reply:
aaah okay, its some react shenanigans with useEffect